lib/mix/tasks/release.clean.ex

```diff
@@ -52,10 +53,9 @@ defmodule Mix.Tasks.Release.Clean do
 
   # Clean release build
   def do_cleanup(:build) do
-    cwd       = File.cwd!
     project   = Mix.Project.config |> Keyword.get(:app) |> Atom.to_string
     version   = Mix.Project.config |> Keyword.get(:version)
-    build     = Path.join([cwd, "_build", "prod"])
+    build     = Mix.Project.build_path
```

Comment:
This change causes `test/plugin_test.exs:5` to fail like:

```
     Assertion with == failed
     code: active == ReleaseManager.Plugin.load_all() |> Enum.sort()
     lhs:  [ReleaseManager.Plugin.Appups, ReleaseManager.Plugin.Consolidation]
     rhs:  []
     stacktrace:
       test/plugin_test.exs:11
```

I am not sure, but explicit specification of the environments seems strange to me.